test/test_examples_3d.jl

```diff
@@ -5,248 +5,32 @@ using Trixi
-  @testset "elixir_mhd_ec.jl" begin
-    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_ec.jl"),
-      l2   = [0.01728564856700721, 0.017770558428794502, 0.01777055842879448, 0.017772303855024686, 0.07402251095395435, 0.010363317528939847, 0.010363317528939873, 0.010365251266968387, 0.00020781240461593321],
-      linf = [0.2648387662456203, 0.33478411844879813, 0.3347841184487984, 0.3698107321074581, 1.2338949711031062, 0.09857295382870013, 0.09857295382870068, 0.10426497383213318, 0.008020325762909617])
-  end
-
-  @testset "elixir_mhd_ec.jl with initial_condition=initial_condition_constant" begin
-    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_ec.jl"),
-      l2   = [5.447723083700851e-16, 2.617532329387129e-15, 3.879181146896324e-15, 2.9617662710806352e-15, 2.684011421945918e-14, 1.371966389577962e-15, 1.3058638553193934e-15, 1.2475529686776005e-15, 1.4573570237027504e-15],
-      linf = [3.9968028886505635e-15, 1.4155343563970746e-14, 2.2898349882893854e-14, 1.8707257964933888e-14, 2.2737367544323206e-13, 1.2878587085651816e-14, 7.327471962526033e-15, 9.325873406851315e-15, 8.568189425129782e-15],
-      atol = 1000*eps(),
-      initial_condition=initial_condition_constant)
-  end
-
-  @testset "elixir_mhd_alfven_wave.jl" begin
-    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_alfven_wave.jl"),
-      l2   = [0.0038734936431132137, 0.0090374494786511, 0.004173235737356234, 0.011605032955462995, 0.006247939160992442, 0.009226153647367723, 0.003460561919838917, 0.011684984122517875, 0.002201096128779557],
-      linf = [0.012630239144833078, 0.03265663914470475, 0.01291368559476544, 0.04444329719972474, 0.027974787995271644, 0.03453507441133391, 0.01022512252706076, 0.04498328542685666, 0.009861640501698054])
-  end
-
-  @testset "elixir_mhd_alfven_wave_mortar.jl" begin
-    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_alfven_wave_mortar.jl"),
-      l2   = [0.0021484230386442484, 0.006826974752320685, 0.0030655172071723945, 0.008736026827927016, 0.005159744590177398, 0.007158578094946651, 0.0028289753083129443, 0.008815093448251163, 0.0022268140033108634],
-      linf = [0.013181862339541217, 0.05433825341881836, 0.02085489712458487, 0.05947341475014939, 0.03171319598340849, 0.05439174028778537, 0.017933784069765202, 0.06036923226300564, 0.012816582427655984],
-      tspan = (0.0, 0.25))
-  end
-
-  @testset "elixir_mhd_orszag_tang.jl" begin
-    @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_mhd_orszag_tang.jl"),
-      l2   = [0.0043911605686459704, 0.041447356655813394, 0.04150129977673066, 0.041503536105048366, 0.03693119824334232, 0.02112559892198415, 0.03295606821170978, 0.03296235617354949, 6.360380099124839e-6],
-      linf = [0.01789383890583951, 0.0848675349327856, 0.08910602912506882, 0.08491879187575965, 0.10444596146695251, 0.05381953967385888, 0.08847783436169666, 0.07784630781912419, 8.236241065117021e-5],
-      tspan = (0.0, 0.06))
-  end
+  include("test_examples_3d_mhd.jl")
```

Comment:
Should we consider also creating an `eulergravity` file (even though it only would contain one test to be... not inconsistent?

Reply:
Okay, Mr. Consistency.

Reply:
:heart:

Reply:
Done in 741cf93.